### PR TITLE
JavaScript Timers Nathan Chon

### PR DESCRIPTION
**Pull request by Nathan-Chon**
animation43

### CODE REVIEW

File: javascript-timers/interval.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+var counter = 0;
+function countdown() {
+  $countdownDisplay.textContent = text[counter];
+  counter++;
+  if (counter >= text.length) {
+    counter = 3;
+    clearInterval();
+  }
+}
+setInterval(countdown, 1000);
```
Comment (robgmerrill) [inline javascript-timers/interval.js:12]: To capture the id we can do the following: 

`var timerId = setInterval(countdown, 1000);`

You will then pass this to the `clearInterval()` function.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Almost! You will need to pass a timer ID to the clearInterval function.

**Comment by Nathan-Chon** [comment]
animation43
Revised version timerId

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**